pkg/controllers/user/multiclusterapp/controller_multiclusterapp.go

```diff
@@ -197,6 +360,26 @@ func (m *MCAppController) createNamespaceAndApp(t *v3.Target, mcapp *v3.MultiClu
 	return t, mcapp, nil
 }
 
+func getAnswerMap(answerMap map[string]map[string]string, projectName string) map[string]string {
+	// find answers for this project, if not found then try finding for the cluster this project belongs to, else finally use the global scoped answer
```

Comment:
can you check if we have this logic in the multi cluster app code? There should be something for merging answers

Reply:
nvm, you just moved this code